assets/sass/style.sass

```
@@ -1,7 +1,7 @@
 {{- $fonts    := site.Params.fonts }}
 {{- $fontSlice := (slice) }}
 {{- range $fonts }}
-{{- $fontSlice = $fontSlice | append (printf "%s:%s" (replace .name " " "+") (delimit .sizes ",")) }}
+{{- $fontSlice := $fontSlice | append (printf "%s:%s" (replace .name " " "+") (delimit .sizes ",")) }}
```

Just checked with Go template syntax. This should really be  `=` (assignment, and not declaration+assignment), otherwise we re-declare the fontSlice var in each iteration. So let's revert here...

```suggestion
{{- $fontSlice = $fontSlice | append (printf "%s:%s" (replace .name " " "+") (delimit .sizes ",")) }}
```

... and fix the transpiler error below in line 13 by quoting the url like so:

```
@import url("{{ $fontsUrl }}")
```

---

I can push a commit.